src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -4,6 +4,7 @@
 using System;
```

Comment:
@SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-linux`
Start-Process.Should handle stdout redirection without error
```powershell
Expected the actual value to be greater than 0, but got 0.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1: line 83
83: 	    $dirEntry.Length | Should -BeGreaterThan 0
```
Start-Process.Should handle arguments with whitespace
```powershell
No such file or directory
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1: line 150
150:         Start-Process -Wait -NoNewWindow testexe.exe -ArgumentList '-echoargs', 'a b', 'c' -RedirectStandardOutput $testdrive/out.txt

```
Start-Process.Should handle arguments with embedded quotes
```powershell
No such file or directory
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1: line 158
158:         Start-Process -Wait -NoNewWindow testexe.exe -ArgumentList '-echoargs', '"a"', 'c' -RedirectStandardOutput $testdrive/out.txt

```
XmlCommand DRT basic functionality Tests.Import-Clixml should work with XML serialization from pwsh.exe
```powershell
Data at the root level is invalid. Line 1, position 1.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1: line 110
110: 		$out = Import-Clixml -Path $testfile

```